<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseConvert</class>
 <widget class="QMainWindow" name="BaseConvert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>158</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1159</width>
    <height>158</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1159</width>
    <height>158</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>29</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="labelHex">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>24</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Hex</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="labelBin">
      <property name="minimumSize">
       <size>
        <width>650</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>24</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Bin</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>29</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="5">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>208</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="3">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>647</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="5">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QCheckBox" name="checkBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Signed</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="labelDec">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>24</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Dec</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>29</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineHex">
      <property name="minimumSize">
       <size>
        <width>190</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>32</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>0x0000000000000000</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="lineBin">
      <property name="minimumSize">
       <size>
        <width>650</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>650</width>
        <height>32</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>0000000000000000000000000000000000000000000000000000000000000000</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QLineEdit" name="lineDec">
      <property name="minimumSize">
       <size>
        <width>211</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>211</width>
        <height>32</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>00000000000000000000</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
